customer/structures.go

```diff
@@ -196,20 +447,30 @@ func (e *Event) File() *File {
 		return nil
 	}
 	var f File
+
```

Comment:
empty line

Reply:
I've added it on purpose, it's present on every specific event cast except File, for example:
- https://github.com/livechat/lc-sdk-go/blob/8e1d0fa3c9a6e6fc9a7dfd2827c4598d80dc2b8d/objects/objects.go#L461-L462
- https://github.com/livechat/lc-sdk-go/blob/8e1d0fa3c9a6e6fc9a7dfd2827c4598d80dc2b8d/objects/objects.go#L360-L361
- https://github.com/livechat/lc-sdk-go/blob/8e1d0fa3c9a6e6fc9a7dfd2827c4598d80dc2b8d/objects/objects.go#L329-L330
- etc.